core/App/screens/ProofRequest.tsx

```diff
@@ -191,15 +190,23 @@ const ProofRequest: React.FC<ProofRequestProps> = ({ navigation, route }) => {
         throw new Error(t('ProofRequest.RequestedCredentialsCouldNotBeFound'))
       }
 
-      const sortedCreds = sortCredentialsForAutoSelect(retrievedCredentials)
```

Comment:
I believe this was added to resolve issues with revoked credentials being selected over valid credentials. Has the issue been resolved in AFJ?

Reply:
Ok, Now I know the test case I will try.

Reply:
Ok, looks like its still required. I've created [this](https://github.com/hyperledger/aries-framework-javascript/issues/1223) ticket in AFJ to propose it gets fixes in AFJ. I'll add it back in.

Reply:
Ok, I added the sort back in for the UI so we can show if we can satisfy the proof with a non-revoked credential. I don't apply it to the results of `autoSelectCredentialsForProofRequest` though because I'm not sure its helpful there.